web/yo/app/scripts/services/storage.js

```diff
@@ -230,7 +249,38 @@ angular.module('oncokbApp')
                 });
                 return deferred.promise;
             };
+            /**
+             *
+             * */
+            self.retrieveMeta = function() {
+                var deferred = $q.defer();
+
+                var retrievePageOfFiles = function(request, result) {
+                    request.execute(function(resp) {
+                        result = result.concat(resp.items);
+                        var nextPageToken = resp.nextPageToken;
+                        if (nextPageToken) {
+                            request = gapi.client.drive.files.list({
+                                q: '"' + config.metaFolderId + '" in parents',
+                                pageToken: nextPageToken,
+                                maxResults: 300
```

Comment:
You don't have to specify the maxResults here. We only look for one document for now.